data/org.fedoraproject.Anaconda.Addons.HelloWorld.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Start the org.fedoraproject.Anaconda.Addons.HelloWorld service.
+# Runs org_fedora_hello_world/service/__main__.py
+
+export PYTHONPATH=/usr/share/anaconda/addons
+
+if [ -f "/usr/libexec/platform-python" ]; then
+    /usr/libexec/platform-python -m org_fedora_hello_world.service
+else
+    python3 -m org_fedora_hello_world.service
```

Comment:
This is unnecessary. I am going to extend `/usr/libexec/anaconda/start-module`, so it can be used by addons as well.

Reply:
https://github.com/rhinstaller/anaconda/pull/2296

Reply:
Perfect, thanks!